CoreService/Workflow.psm1

```diff
@@ -0,0 +1,231 @@
+#Requires -version 3.0
+
+<#
+**************************************************
+* Private members
+**************************************************
+#>
+
+<#
+**************************************************
+* Public members
+**************************************************
+#>
+
+function Get-TridionWorkflowItems
+{
+    <#
+    .Synopsis
+    Gets a list of Workflow Items in Tridion Content Manager By Status or AssignedTo Filter
+    .Description
+    Gets a list of Workflow Items in Tridion Content Manager By Status or AssignedTo Filter
+    .Notes
+    Example of properties available: Id, Title Etc.
+    
+    For a full list, consult the Content Manager Core Service API Reference Guide documentation 
+    (Tridion.ContentManager.Data.CommunicationManagement.ProcessInstanceData object)
+    .Inputs
+     [string] Status(Not Mandetory): Workflow ProcessInstanceData by status "Active" or "Hstoricle" (Bydefault is will show all the data)
```

Comment:
Some typos here.

Reply:
OK. Changes done. Will update in Next Commit